app/src/main/java/org/oppia/app/player/content_interaction/ContentInteractionFragmentPresenter.kt

```diff
@@ -0,0 +1,147 @@
+    interactionInstanceId = "MultipleChoiceInput"
+
+    // Todo(veena): Keep the below code for actual implementation
+    val gaeCustomizationArgs: Any? = gaeCustomizationArgsMap!!.get("choices")?.value
+
+    if (interactionInstanceId.equals("MultipleChoiceInput")) {
+      val gaeCustomArgsInString: String = gaeCustomizationArgs.toString().replace("[", "").replace("]", "")
+      var items = gaeCustomArgsInString.split(",").toTypedArray()
+      addRadioButtons(items)
+    } else if (interactionInstanceId.equals("ItemSelectionInput") || interactionInstanceId.equals("SingleChoiceInput")) {
+      val gaeCustomArgsInString: String = gaeCustomizationArgs.toString().replace("[", "").replace("]", "")
+      var items = gaeCustomArgsInString.split(",").toTypedArray()
+      addCheckbox(items)
+    } else {
+      //Do no show any view
+    }
+  }
+
+  fun addCheckbox(optionsArray: Array<String>) {
+    for (row in 0..0) {
+      val ll = LinearLayout(context)
+      ll.orientation = LinearLayout.VERTICAL
+
+      for (i in 0..optionsArray.size - 1) {
+        val cb = CheckBox(context)
+        cb.id = View.generateViewId()
+        cb.text = convertHtmlToString(optionsArray[i], cb).toString()
+
+        cb.setOnCheckedChangeListener(CompoundButton.OnCheckedChangeListener { buttonView, isChecked ->
+          val msg = "You have " + (if (isChecked) "checked" else "unchecked") + " this Check it Checkbox."
+          Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
+        })
+        
+        ll.addView(cb)
+      }
+      binding.root.radioGroup.addView(ll)
+    }
+  }
+
+  fun addRadioButtons(optionsArray: Array<String>) {
+    for (row in 0..0) {
+      val rg = RadioGroup(context)
+      rg.orientation = LinearLayout.VERTICAL
+
+      for (i in 0..optionsArray.size - 1) {
+        val rdbtn = RadioButton(context)
+        rdbtn.id = View.generateViewId()
+        rdbtn.text = convertHtmlToString(optionsArray[i], rdbtn).toString()// + rdbtn.id
+        rdbtn.gravity = (Gravity.RIGHT); (Gravity.CENTER)
+
+        rg.setOnCheckedChangeListener(object : RadioGroup.OnCheckedChangeListener {
+          override fun onCheckedChanged(group: RadioGroup, checkedId: Int) {
+            for (i in 0 until group.childCount) {
+              val btn = group.getChildAt(i) as RadioButton
+              if (btn.id == checkedId) {
+                val text = btn.text
+                Toast.makeText(context, "" + text, Toast.LENGTH_LONG).show()
+                return
+              }
+            }
+
+          }
+        })
+        rg.addView(rdbtn)
+      }
+      binding.root.radioGroup.addView(rg)
+    }
+  }
+
+  fun convertHtmlToString(rawResponse: Any?, rdbtn: View): Spanned {
+
+    var htmlContent = rawResponse as String;
+    var result: Spanned
+    var CUSTOM_TAG = "oppia-noninteractive-image"
+    var HTML_TAG = "img"
+    var CUSTOM_ATTRIBUTE = "filepath-with-value"
+    var HTML_ATTRIBUTE = "src"
```

Comment:
I think all these variables can be converted to val. But better you define all these as `const val` before above the entire-class-code